src/app/routes/connector-ui/asset-page/asset-edit-dialog/asset-edit-dialog.component.html

```diff
@@ -39,6 +53,13 @@ <h1 mat-dialog-title>
               matInput
               autocomplete="new-id"
               [formControl]="ctrl" />
+            <button
+              class="!scale-[0.9]"
+              mat-icon-button
+              matSuffix
+              matTooltip="Asset ID will be auto-generated in URL format, prefixed by urn:artifact:. It can be changed by user even though CaaS uses the Name of Asset as default Asset ID">
```

Comment:
Asset ID used internally. Generated from the asset name by default, must be prefixed by urn:artifact:

or so, idk. But the wording before was too long and not really helpful. What is a "URL format"?